README.md

```diff
@@ -178,6 +179,13 @@ Same goes for deleted headings, these are cleared out.
 
 Updating the TOC can also be accomplished without saving by picking from the menu: Tools > MarkdownTOC > Update TOC
 
+
+#### Supported file extensions
+
+Make sure your file's extension is in following list.
```

Comment:
Hi @naokazuterada 

You just missed an article (the):

"Make sure our file's extension is in the following list."

Reply:
Oh, it was certainly missing...
Thank you! I will add 'the' and merge it later.